### PR TITLE
Safe browsing quick fixes

### PR DESCRIPTION
**Pull request by dennisreimann**
As discussed in #2139 these changes might address the issue.

### CODE REVIEW

File: BTCPayServer/Controllers/AccountController.cs [hunk 1]
```diff
@@ -66,6 +66,7 @@ public string ErrorMessage
 
         [HttpGet]
         [AllowAnonymous]
+        [Route("/login")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:69]:
```suggestion
        [Route("~/login")]
```

File: BTCPayServer/Controllers/AccountController.cs [hunk 2]
```diff
@@ -89,6 +90,7 @@ public async Task<IActionResult> Login(string returnUrl = null, string email = n
 
         [HttpPost]
         [AllowAnonymous]
+        [Route("/login")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:93]:
```suggestion
        [Route("~/login")]
```

File: BTCPayServer/Controllers/AccountController.cs [hunk 3]
```diff
@@ -396,6 +398,7 @@ public IActionResult Lockout()
 
         [HttpGet]
         [AllowAnonymous]
+        [Route("/register")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:401]:
```suggestion
        [Route("~/register")]
```

File: BTCPayServer/Controllers/AccountController.cs [hunk 4]
```diff
@@ -413,6 +416,7 @@ public async Task<IActionResult> Register(string returnUrl = null, bool logon =
 
         [HttpPost]
         [AllowAnonymous]
+        [Route("/register")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:419]:
```suggestion
        [Route("~/register")]
```

File: BTCPayServer/Controllers/AccountController.cs [hunk 5]
```diff
@@ -479,6 +483,7 @@ public async Task<IActionResult> Register(RegisterViewModel model, string return
         public bool RegisteredAdmin { get; set; }
 
         [HttpGet]
+        [Route("/logout")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:486]:
```suggestion
        [Route("~/logout")]
```

File: BTCPayServer/Controllers/AccountController.cs [hunk 6]
```diff
@@ -527,13 +532,15 @@ public async Task<IActionResult> ConfirmEmail(string userId, string code)
 
         [HttpGet]
         [AllowAnonymous]
+        [Route("/forgot-password")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:535]:
```suggestion
        [Route("~/forgot-password")]
```

File: BTCPayServer/Controllers/AccountController.cs [hunk 7]
```diff
@@ -527,13 +532,15 @@ public async Task<IActionResult> ConfirmEmail(string userId, string code)
...
         public IActionResult ForgotPassword()
         {
             return View();
         }
 
         [HttpPost]
         [AllowAnonymous]
+        [Route("/forgot-password")]
```
Comment (Kukks) [inline BTCPayServer/Controllers/AccountController.cs:543]:
```suggestion
        [Route("~/forgot-password")]
```

File: BTCPayServer/wwwroot/modal/btcpay.js [hunk 8]
```diff
@@ -39,6 +38,8 @@
     iframe.style.height = '100%';
     iframe.style.width = '100%';
     iframe.style.zIndex = '2000';
+    // Removed, see https://github.com/btcpayserver/btcpayserver/issues/2139#issuecomment-768223263
+    // iframe.setAttribute('allowtransparency', 'true');
```
Comment (Kukks) [inline BTCPayServer/wwwroot/modal/btcpay.js:42]: What are the implications for this? Is there potential for it to render badly?

File: BTCPayServer/Controllers/AccountController.cs (hunk 6 above)
Comment (Kukks) [reply BTCPayServer/Controllers/AccountController.cs:535]: We need to leave the old route too. If there are active password resets, their links will be broken once they update.

File: BTCPayServer/Controllers/AccountController.cs (hunk 7 above)
Comment (Kukks) [reply BTCPayServer/Controllers/AccountController.cs:543]: We need to leave the old route too. If there are active password resets, their links will be broken once they update.

**Review by Kukks — CHANGES_REQUESTED** [review]
Please prepend `~` to the route as it ensures that the controller doesnt do any funny business and try to itself  as part of the route.

Also, the forgot password route change will break existing links

Meh, hate these blind changes but will hopefully give us some insight.. (I messaged netcraft but still no response btw)

File: BTCPayServer/wwwroot/modal/btcpay.js (hunk 8 above)
Comment (dennisreimann) [reply BTCPayServer/wwwroot/modal/btcpay.js:42]: I haven't noticed any difference in Firefox.

Comment (NicolasDorier) [reply BTCPayServer/wwwroot/modal/btcpay.js:42]: I think @rockstardev wrote this, he might have an idea.

**Comment by NicolasDorier** [comment]
Another commit I would like to see: Remove the Tor link from the login/register page. This is not needed anymore.

**Comment by dennisreimann** [comment]
@NicolasDorier see 4984674b1d23b6978ddedacc21371313d983ba6f

**Review by Kukks — APPROVED** [review]
ack

**Comment by NicolasDorier** [comment]
I will make a release with only those fixes.

**Pull request merged**